client/package.json

```diff
@@ -110,7 +110,11 @@
       "ts-jest": {
         "tsConfig": "tsconfig.spec.json"
       }
-    }
+    },
+    "modulePathIgnorePatterns": [
+      "<rootDir>/dist/",
+      "<rootDir>/dist_electron/"
+    ]
```

Comment:
On linux, `build:electron` creates a `.snap` package in `dist_electron`, which is the same file extension of a jest snapshot.  The snapshot doesn't match and so tests fail.